validate.go

```diff
@@ -192,6 +194,48 @@ func checkProcess(process rspec.Process, rootfs string) {
 	}
 }
 
+func supportedMountTypes() (map[string]bool, error) {
+	supportedTypes := make(map[string]bool)
+	f, err := os.Open("/proc/filesystems")
+	if err != nil {
+		return supportedTypes, err
+	}
+	defer f.Close()
+
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return supportedTypes, err
+		}
+
+		text := s.Text()
+		parts := strings.Split(text, "\t")
+		if len(parts) > 1 {
+			supportedTypes[parts[1]] = true
+		} else {
+			supportedTypes[parts[0]] = true
+		}
+	}
+
+	supportedTypes["nfs"] = true
+	supportedTypes["bind"] = true
```

Comment:
While I agree that bind mounts are important, I don't see any wording in [the current spec][1] about them.  We probably want to file a PR adding them as an additional “type” (and maybe add other wording about bind(8) options, e.g. the shared/slave/private/unbindable stuff?) before allowing them here.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config.md#mounts